AsyncDisplayKit/ASTextNode.mm

```diff
@@ -404,7 +416,7 @@ - (CGSize)calculateSizeThatFits:(CGSize)constrainedSize
 
   [self setNeedsDisplay];
   
-  CGSize size = [self _renderer].size;
+  CGSize size = [self _rendererWithBoundsSlow:{.size=constrainedSize}].size;
```

Comment:
Because we already call _invalidateRenderer above if the constrainedSize has changed, could you add a comment above this line with a quick reminder about why this has to call the Slow version?

I know you've written a detailed description of this in a PR or Issue that we reviewed recently - perfect to bring some of that into the code here.  Even if it ends up being pretty long, I think it's key context to have in the code, and when we optimize with a more advanced technique we can remove it.